content/docs/command-reference/add.md

```diff
@@ -38,9 +38,9 @@ each one:
 
 1. Calculate the file hash.
 2. Move the file contents to the cache (by default in `.dvc/cache`), using the
-   file hash to form the cached file path. (See
-   [Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
-   for more details.)
+   file hash to form the cached file path. (See [Structure of cache directory](/
+   doc/user-guide/dvc-files-and-directories#structure-of-cache-directory) for
```

Comment:
This is a broken link. Do you know why/how was this format changed @utkarshsingh99 ? Did you check how it looks running the app locally?

Reply:
Not really sure about this. I didn't update this manually.
Left it here thinking it must be a styling fix. 
Anyway, fix the newline error. It was including the '\n' in the url too.

Reply:
> must be a styling fix

The formatter would never do that, it was probably a typo, no worries. But if you noticed it you shouldn't have assumed it was fine.